lib/signaling/v2/peerconnection.js

```diff
@@ -1159,4 +1183,46 @@ function shouldStopTransceiver(sdp, transceiver) {
   return false;
 }
 
+/**
+ * Perform certain updates after an SDP negotiation is completed.
+ * @param {PeerConnectionV2} pcv2
+ * @returns {void}
+ */
+function negotiationCompleted(pcv2) {
+  if (pcv2._isRTCRtpSenderParamsSupported) {
```

Comment:
@makarandp0 I don't think we should, because there would be no way to know if we have to trigger the offer <--> answer process for cases where the API is not supported.